packages/core/src/browser/keybinding.ts

```diff
@@ -29,6 +29,7 @@ import * as common from '../common/keybinding';
 
 export enum KeybindingScope {
     DEFAULT,
+    PLUGIN,
```

Comment:
We should not call it `PLUGIN`, other packages should not be aware of them. The plugin system is optional.

Let's call it `DEFAULT_OVERRIDING` to point out that they are default but allow to collide.

Reply:
done